elements/pfe-dropdown/src/_shared-assets.scss

```
@@ -0,0 +1,8 @@
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: dropdown;
+
+$variables: ( 
+  SpacingVertical: calc(#{pfe-var(container-spacer)} * 0.5),
```

```suggestion
  SpacingVertical: calc(#{pfe-var(container-padding)} * 0.5),
```

Subtle difference, but the spacer is intended for space between items in a container. The padding is the space around the edges of the container. Check out [this discussion](https://github.com/patternfly/patternfly-elements/issues/675) if interested. :)